### PR TITLE
Added initial spring boot integration

### PR DESCRIPTION
**Pull request by ghost**
Rendering structure still to be done

### CODE REVIEW

File: src/main/java/evolvio/Application.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.ConfigurableApplicationContext;
+
+@SpringBootApplication
+public class Application {
+    public static void main(String[] args) {
+        SpringApplicationBuilder builder = new SpringApplicationBuilder(Application.class);
+        builder.headless(false);
+        ConfigurableApplicationContext context = builder.run(args);
```
Comment (Holkin) [inline src/main/java/evolvio/Application.java:12]: public static void main(String[] args) {
        new SpringApplicationBuilder(Application.class).headless(false).run(args);
    }

File: src/main/java/evolvio/render/WindowStarter.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package evolvio.render;
```
Comment (Holkin) [inline src/main/java/evolvio/render/WindowStarter.java]: What about evolviol.ui.view package?
We may need evolviol.ui.listener in future.
Entities rendering can be moved to evolviol.ui.domain or something like this.

File: src/main/java/evolvio/render/WindowStarter.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+@Service
+public class WindowStarter {
+    @Autowired
+    public WindowStarter(@Value("${width}") int width, @Value("${height}") int height) {
+        JFrame frame = new JFrame();
+        frame.setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
+        frame.setSize(width, height);
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
```
Comment (Holkin) [inline src/main/java/evolvio/render/WindowStarter.java]: frame.setTitle("Evolution simulator");

**Review by Holkin — COMMENTED** [review]
resources/banner.txt would be nice
```
  ______          _       _   _                _____ _                 _       _
 |  ____|        | |     | | (_)              / ____(_)               | |     | |
 | |____   _____ | |_   _| |_ _  ___  _ __   | (___  _ _ __ ___  _   _| | __ _| |_ ___  _ __
 |  __\ \ / / _ \| | | | | __| |/ _ \| '_ \   \___ \| | '_ ` _ \| | | | |/ _` | __/ _ \| '__|
 | |___\ V / (_) | | |_| | |_| | (_) | | | |  ____) | | | | | | | |_| | | (_| | || (_) | |
 |______\_/ \___/|_|\__,_|\__|_|\___/|_| |_| |_____/|_|_| |_| |_|\__,_|_|\__,_|\__\___/|_|


```

**Review by quete12 — APPROVED** [review]
Looks good

File: src/main/java/evolvio/Application.java (hunk 1 above)
Comment (ghost) [reply src/main/java/evolvio/Application.java:12]: Not sure what this adds

**Pull request merged**